modules/nf-core/sratools/prefetch/main.nf

```diff
@@ -32,4 +32,16 @@ process SRATOOLS_PREFETCH {
     }
 
     template 'retry_with_backoff.sh'
+
+    stub:
+    """
+    mkdir $id
```

Comment:
Again, I like to echo the original command.

Reply:
`prefetch` is using a template for script. Is there a way to echo the template?

Reply:
My bad, I had forgotten about that.